setup.py

```diff
@@ -24,7 +24,7 @@
 
 
 package_name = "dbt"
-package_version = "0.21.0a1"
+package_version = "0.20.0"
```

Comment:
Could you switch this back? I'm guessing you changed to test with current plugin versions